satellite/console/mail.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package console
+
+import (
+	"net/mail"
+)
+
+const (
+	// ActivationSubject activation email subject
+	ActivationSubject = "Activate your email"
+	// InvitationSubject invitation email subject
+	InvitationSubject = ""
+	// ForgotPasswordSubject forgot password email subject
+	ForgotPasswordSubject = ""
+)
+
+// MailTemplate is implementation of satellite/mailservice.Template interface
+type MailTemplate struct {
+	to            mail.Address
+	subject       string
+	htmlPath      string
```

Comment:
Mail templates shouldn't care about the full path of the templates. It makes constructing them more complicated. Let `mailservice` handle that concern.